ldms/python/ldmsd/ldmsd_controller

```diff
@@ -357,42 +357,66 @@ class LdmsdCmdParser(cmd.Cmd):
     def complete_daemon_exit(self, text, line, begidx, endidx):
         return self.__complete_attr_list('daemon_exit', text)
 
+    def __handle_reconnect_alias(reconnect_interval, reconnect, interval):
```

Comment:
Could we possibly move this function to ldmsd_communicator.py? I realize we don't have parser_util.py merged yet, or I'd suggest moving it there. This would be useful to be able to call when parsing YAML files for both the ldmsd -y option and maestro_ctrl

Reply:
@nick-enoent  let's look at the API and decide if we need something else.